packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -126,6 +125,13 @@ function ListView<T extends object>(props: SpectrumListProps<T>, ref: DOMRef<HTM
       preview
     });
   }
+  let layout = useListLayout(
+    state,
+    props.density || 'regular',
+    // !!0 is false, so we can cast size or undefined and they'll be falsy
+    state.selectionManager.disabledBehavior === 'selection' || !!dragState?.draggingKeys.size,
+    overflowMode);
+
```

Comment:
Weird, I get double drop indicators when I start dragging via keyboard in the reorder story:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/9661127/172733574-0b5e2e7c-8c69-455b-9c99-5da55bda84c7.png">
I think it is something with the ListLayout and it getting a new instance due to the drag state change, I noticed the Item 2 row is 47px vs 40px for the other rows